user/views.py

```diff
@@ -0,0 +1,60 @@
+import json
+import re
+import bcrypt
+
+from django.http                       import JsonResponse, HttpResponse
+from django.views                      import View
+
+from user.models                       import (
+    User
+)
```

Comment:
- 큰 문제가 되는것은 아니지만 import 문 기준으로 정렬해주실 때 굳이 저렇게 space 를 많이주실 필요없이 해당 block 에서 가장 긴 from 문 기준으로 잡아주시면 깔끔합니다! 
- 그리고 아래 user.models import 문 구분 잘 해주셨는데 현재 모델 class를 하나만 import 해오는 과정이니 굳이 괄호를 추가해 아래로 정렬해주실 필요까지 없을 것 같아 아래와 같이 수정해봤습니다.
```suggestion
from django.http  import JsonResponse, HttpResponse
from django.views import View

from user.models import User
```

Reply:
넵! 수정하겠습니다 :)